Tests/UnitTests/Core/TrackFitting/GsfTests.cpp

```diff
@@ -127,45 +127,45 @@ BOOST_AUTO_TEST_CASE(ZeroFieldNoSurfaceForward) {
   auto options = makeDefaultGsfOptions();
 
   tester.test_ZeroFieldNoSurfaceForward(gsfZero, options, multi_pars, rng, true,
-                                        true);
+                                        false);
```

Comment:
what is the reason for setting `expected_smoothed` to `false`?

Reply:
That we do not smooth anymore. The whole thing is a little bit more complicated with the GSF: Smoothing means here that we combine the forward and the backward multi-component state to a smoothed multi-component state, that represents best the state at the current surface.

However, this is both time consuming and we also do not return the multicomponent states anyway in the result, so I decided to remove this code.